README.md

```diff
@@ -6,288 +6,324 @@ Code to generate and deploy the dashboard for the Cumulus API.
 
-#### Build the dashboard to be served by the Cumulus API.
+### Run the dashboard locally via Docker Image (users/operators)
 
-With the Cumulus API it is possible to [serve the dashboard from an s3 Bucket](https://nasa.github.io/cumulus-api/#serve-the-dashboard-from-a-bucket).  If you wish to do this, you must build the dashboard with the environment variable `SERVED_BY_CUMULUS_API` set to `true`.  This configures the dashboard to work from the Cumulus `dashboard` endpoint.  This option should be considered when you can't serve the dashboard from behind CloudFront, for example in NGAP sandbox environments.  If you wish to serve the dashboard from behind [CloudFront](https://aws.amazon.com/cloudfront/).  Build a `dist` with your configuration for `APIROOT` and omitting `SERVED_BY_CUMULUS_API` and follow the cumulus operator docs on [serving the dashboard from CloudFront](https://nasa.github.io/cumulus/docs/next/operator-docs/serve-dashboard-from-cloudfront).
+Once you have a built a dashboard and the contents are in the `./dist` directory, you can create a docker container that will serve the dashboard behind a simple nginx configuration. Having a runnable Docker image is useful for testing a build before deployment or for NGAP Sandbox environments, where if you configure your computer to [access Cumulus APIs via SSM](https://wiki.earthdata.nasa.gov/display/CUMULUS/Accessing+Cumulus+APIs+via+SSM), you can run the dashboard container locally against the live Sandbox Cumulus API.
 
-The compiled files will be placed in the `dist` directory.
+The script `./bin/build_dashboard_image.sh` takes a pre-built dashboard in the `./dist` directly and packages it in a Docker container behind a basic nginx configuration. The script takes one optional parameter, the tag to name the generated image which defaults to cumulus-dashboard:latest.
 
-### Building a specific dashboard version
+Example of building and running the project in Docker:
+```bash
+  $ ./bin/build_dashboard_image.sh cumulus-dashboard:production-1
+```
 
-`cumulus-dashboard` versions are distributed using tags in GitHub. You can pull a specific version in the following manner:
+That command builds a Docker image with the name `cumulus-dashboard` and tag `production-1`. This image can be run in Docker to serve the Dashboard.
 
 ```bash
-  $ git clone https://github.com/nasa/cumulus-dashboard
-  $ cd cumulus-dashboard
-  $ git fetch origin ${tagNumber}:refs/tags/${tagNumber}
-  $ git checkout ${tagNumber}
+  $ docker run --rm  -p 3000:80 cumulus-dashboard:production-1
 ```
 
-Then follow the steps noted above to build the dashboard locally or using Docker.
+In this example, the dashboard would be available at `http://localhost:3000/` in any browser.
+
+To stop the container, find the container id with `docker ps`.  In our exaple it will be a container with NAME `cumulus-dashboard` and TAG `production-1`, then running `docker stop <containerID>` will stop the docker container.
```

Comment:
It is if I don't figure out how to make it stop easier tomorrow.
So If I tell them to run it in interactive mode, it will stop with ^C
I will change the example line to do that and then remove these instructions.
` $ docker run --rm -it -p 3000:80 cumulus-dashboard:production-1`
This is really just how docker works, and we shouldn't have to teach that too much.

Reply:
I went with updating the example and removing the Docker instructions.

Reply:
It's gone now